src/css/styles.less

```diff
@@ -0,0 +1,19 @@
+@JW_CSS_SMOOTH_EASE: opacity .25s ease;
+
+.jwplayer {
+    position: relative;
```

Comment:
For some reason `position: 'relative'` is not getting set, making the player fill the page.